### PR TITLE
Add time measure to quality evaluation

### LINKED ISSUE #81: Quality metrics with time

### PR DESCRIPTION
**Pull request by HafnerMichael**
closes #81

### CODE REVIEW

**Review by Xetiic — APPROVED** [review]

**Comment by HafnerMichael** [comment]
@m-langer please do not forget to run the quality script and push the results and plots

File: scripts/preprocessing.py [hunk 1]
```diff
@@ -27,25 +27,25 @@ async def preprocess(cropNeeded=False,
...
             executor.submit(jxrenc_2.encode_jxr, printProgress, maxFileSizeKb),
-            executor.submit(webP.encode_webp, printProgress, maxFileSizeKb),
-            executor.submit(jpegxl.encode_jpgxl, printProgress, maxFileSizeKb),
-            executor.submit(jpeg2000enc.encode_jpeg2k, printProgress, maxFileSizeKb),
-            executor.submit(jpegenc.encode_jpeg, printProgress, maxFileSizeKb),
+            #executor.submit(webP.encode_webp, printProgress, maxFileSizeKb),
+            #executor.submit(jpegxl.encode_jpgxl, printProgress, maxFileSizeKb),
+            #executor.submit(jpeg2000enc.encode_jpeg2k, printProgress, maxFileSizeKb),
+            #executor.submit(jpegenc.encode_jpeg, printProgress, maxFileSizeKb),
         ]
```
Comment (HafnerMichael) [inline scripts/preprocessing.py:40]: please remove the comments

Comment (m-langer) [reply scripts/preprocessing.py]: removed comments

**Review by m-langer — APPROVED** [review]

**Review by AleksandarRa — COMMENTED** [review]
looks good

**Review by AleksandarRa — APPROVED** [review]

**Pull request merged**